src/ctree-app/ctree-landing.html

```diff
@@ -77,6 +78,8 @@
         display: block;
         --app-primary-color: #5F9F00;
         --app-secondary-color: black;
+        --app-header-padding-fix: 65px;
+        --app-header-margin-fix: -65px;
```

Comment:
Glad to see you added this.  Did you check to see if there are any other places where these could be used?

I make it a habit when I'm improving a practice like, adding a named constant/variable, to look for other usages instead of leaving it for people to add piecemeal.  Not saying you didn't/don't, just raising it just in case.

Reply:
> Did you check to see if there are any other places where these could be used?

No, I did not. The only other border that I noticed was this one:

![other_border](https://user-images.githubusercontent.com/44947175/70097043-523e6d00-15dd-11ea-9307-4acce0ce0f1c.PNG)

But this could be given a different variable name and number of pixels to adjust since they are not one and the same.